tests/components/cloudflare/test_init.py

```diff
@@ -0,0 +1,58 @@
+"""Test the Cloudflare integration."""
+from pycfdns.exceptions import CloudflareConnectionException
+
+from homeassistant.components.cloudflare.const import DOMAIN, SERVICE_UPDATE_RECORDS
+from homeassistant.config_entries import (
+    ENTRY_STATE_LOADED,
+    ENTRY_STATE_NOT_LOADED,
+    ENTRY_STATE_SETUP_RETRY,
+)
+
+from . import ENTRY_CONFIG, init_integration
+
+from tests.common import MockConfigEntry
+
+
+async def test_unload_entry(hass, cfupdate):
+    """Test successful unload of entry."""
+    entry = await init_integration(hass)
+
+    assert len(hass.config_entries.async_entries(DOMAIN)) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)